<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SALevelClearCount</class>
 <widget class="QWidget" name="SALevelClearCount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>36</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxMain">
   <item>
    <layout class="QHBoxLayout" name="hboxCharacter"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLevels">
     <item row="0" column="0">
      <widget class="QLabel" name="lblLevelName">
       <property name="text">
        <string>Level:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblSonic">
       <property name="text">
        <string>Sonic:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblUnused1">
       <property name="text">
        <string>Unused:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lblTails">
       <property name="text">
        <string>Tails:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="lblKnuckles">
       <property name="text">
        <string>Knuckles:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="lblUnused2">
       <property name="text">
        <string>Unused:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="lblAmy">
       <property name="text">
        <string>Amy:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="lblGamma">
       <property name="text">
        <string>Gamma:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="lblBig">
       <property name="text">
        <string>Big:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="20" column="0" colspan="9">
      <spacer name="vspcSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
